Debonair-Squids/Bangazon_WebApp
# LG_productType

# Steps To Test
1. Git fetch --all
2. checkout LG_productType
3. there should be a migration called "20180219172252_Initial.cs"
4. make sure all of our tables are there

[Hyperlink to ticket](http://www.google.fr/)

### Description of what should happen when steps are followed

**At least two people needs to approve pull request before merged to master**
